src/main/resources/messages.properties

```diff
@@ -41,6 +41,7 @@ invalid.category.sizes.id=Sizes are available only for the categories "Clothes"
 
 user.not-owner=User don't own gained advertisement:
 user.created=New user has been created
+invalid.category.id=Category has to exist by id, but it does not with this id: 
```

Comment:
I think you should add this message also in file message_ua

Reply:
Added